x-pack/plugins/enterprise_search/common/types/indices.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export interface ElasticsearchIndex {
```

Comment:
I can update it, I just reused the same payload schema from the index_management plugin, personally it may be better actually to stay with the same shape with ES API.

Reply:
I updated but to make it easier to use on frontend I return it as an array instead.

Reply:
👍 Thanks!